src/elements/callout/calloutDataTransformer.ts

```diff
@@ -38,14 +59,37 @@ export const transformElementOut: TransformOut<
   isNonCollapsible,
   campaignId,
   tagId,
+  useDefaultPrompt,
+  overridePrompt,
+  useDefaultTitle,
+  overrideTitle,
+  useDefaultDescription,
+  overrideDescription,
 }: FieldNameToValueMap<typeof calloutFields>): ExternalCalloutData => {
+  const cleanupDescription = (desc: string): string => {
+    // Empty richtext fields have a paragraph tag, so we should check if its empty
+    const noTags = desc.replace(/(<([^>]+)>)/gi, "");
+    return noTags.trim() ? desc : "";
+  };
+
+  const getOptionalFields = () => ({
```

Comment:
Any reason not to do this inline? (I'm super happy either way, I'm just curious as to your reasoning, perhaps b/c it makes the intent clearer?)